lib/versioning/rez/index.spec.ts

```diff
@@ -257,23 +257,23 @@ describe('versioning/rez/index', () => {
     ${'<=1.3'}           | ${'widen'}    | ${'1.2.3'}     | ${'2.2.5'} | ${'<=2.2.5'}
     ${'<=1.2.3'}         | ${'widen'}    | ${'1.2.3'}     | ${'2.2.5'} | ${'<=2.2.5'}
     ${'<8'}              | ${'replace'}  | ${'7.2.3'}     | ${'8.2.5'} | ${'<9'}
-    ${'<7.3'}            | ${'replace'}  | ${'7.2.3'}     | ${'8.2.5'} | ${'<9.0'}
+    ${'<7.3'}            | ${'replace'}  | ${'7.2.3'}     | ${'8.2.5'} | ${'<9'}
```

Comment:
```suggestion
    ${'<7.3'}            | ${'replace'}  | ${'7.2.3'}     | ${'8.2.5'} | ${'<8.3'}
```

Retain same precision